dev-env/lib/dade-dump-profile

```diff
@@ -20,34 +20,42 @@ unset NIX_PATH
 set -Eeuo pipefail
 
 if [[ -n "${DADE_DEBUG+x}" ]]; then
-    set -x
+  set -x
 fi
 
 errcho() {
-    >&2 echo "[dev-env] $*"
+  >&2 echo "[dev-env] $*"
+}
+
+get_nix_version() {
```

Comment:
Can we just require 2.4 instead of introducing versioning logic?

Reply:
Works for me! I will update the docs and the version check.